src/harness/loggedIO.ts

```diff
@@ -223,7 +223,18 @@ namespace Playback {
                 recordLog.directoriesRead.push(logEntry);
                 return result;
             },
-            (path, extension, exclude) => findResultByPath(wrapper, replayLog.directoriesRead.filter(d => d.extension === extension && ts.arrayIsEqualTo(d.exclude, exclude)), path));
+            (path, extension, exclude) => findResultByPath(wrapper,
+                    replayLog.directoriesRead.filter(
+                        (d) => {
+                            if (d.extension === extension) {

```

Comment:
we have multiple reads to the same extension but with different excludes in the same session?